lib/migration.pm

```diff
@@ -36,7 +36,6 @@ sub setup_sle {
     select_console 'root-console';
 
     if (is_ppc64le && is_sle('<=12-sp5')) {
```

Comment:
could you please check if we can remove the whole condition, I think after the bug is fixed, we don't even need to restart the service. But you will need at least 10 runs, because I remember it was a race condition.

Reply:
So it is not a workaround for bug, just a test issue, I don't think we need remove the condition since other arches do not have this issue.

Reply:
I'm proposing remove the condition and the workaround and the soft-failure, remove all, because I believe this is already fixed.

Reply:
@skliu could you please paste in your first PR comment those 10 runs to evaluate?

Reply:
Done.